### ISSUE TITLE
Writer function for results

### ISSUE THREAD

**Issue by AlessandroVol23** [AlessandroVol23/predict_ico_success_lmu#12]
We should save all results we get from our cross validation set with additional fields like:
- Features (all, feature set 1, feature set 2, etc.)
- Hyperparameter for model (n_estimators, min_split, etc.) 

And concat all of that into one csv or json file
